backend/src/test/java/com/votogether/domain/category/service/CategoryServiceTest.java

```diff
@@ -0,0 +1,133 @@
+    MemberRepository memberRepository;
+
+    @Test
+    @DisplayName("모든 카테고리를 가져온다.")
+    void getAllCategories() {
+        // given
+        Category category = Category.builder()
+                .name("개발")
+                .build();
+
+        categoryRepository.save(category);
+
+        // when
+        List<CategoryResponse> categories = categoryService.getAllCategories();
+
+        // then
+        assertAll(
+                () -> assertThat(categories.get(0).id()).isNotNull(),
+                () -> assertThat(categories.get(0).name()).isEqualTo("개발"),
+                () -> assertThat(categories.get(0).isFavorite()).isFalse()
+        );
+    }
+
+    @Test
+    @DisplayName("선호하는 카테고리를 선호 카테고리 목록에 추가한다.")
+    void addFavoriteCategory() {
+        // given
+        Category category = Category.builder()
+                .name("개발")
+                .build();
+
+        Member member = Member.builder()
+                .gender(Gender.MALE)
+                .point(0)
+                .socialType(SocialType.GOOGLE)
+                .nickname("user1")
+                .socialId("kakao@gmail.com")
+                .birthDate(
+                        LocalDateTime.of(1995, 7, 12, 0, 0))
+                .build();
+
+        categoryRepository.save(category);
+        memberRepository.save(member);
+
+        Long categoryId = category.getId();
+
+        // when
+        categoryService.addFavoriteCategory(member, categoryId);
+
+        // then
+        MemberCategory memberCategory = memberCategoryRepository.findByMemberAndCategory(member, category).get();
+
+        Assertions.assertAll(
+                () -> assertThat(memberCategory.getMember()).isSameAs(member),
+                () -> assertThat(memberCategory.getCategory()).isSameAs(category)
+        );
+    }
+
+    @Test
+    @DisplayName("선호하는 카테고리를 선호 카테고리 목록에 삭제한다.")
+    void removeFavoriteCategory() {
+        // given
+        Category category = Category.builder()
+                .name("개발")
+                .build();
+
+        Member member = Member.builder()
+                .gender(Gender.MALE)
+                .point(0)
+                .socialType(SocialType.GOOGLE)
+                .nickname("user1")
+                .socialId("kakao@gmail.com")
+                .birthDate(
+                        LocalDateTime.of(1995, 7, 12, 0, 0))
+                .build();
+
+        MemberCategory memberCategory = MemberCategory.builder()
+                .member(member)
+                .category(category)
+                .build();
+
+        categoryRepository.save(category);
+        memberRepository.save(member);
+        memberCategoryRepository.save(memberCategory);
+
+        Long categoryId = category.getId();
+
+        // when
+        categoryService.removeFavoriteCategory(member, categoryId);
+
+        // then
+        boolean isEmpty = memberCategoryRepository.findByMemberAndCategory(member, category).isEmpty();
+        assertThat(isEmpty).isTrue();
```

Comment:
```suggestion
        Optional<MemberCategory> findedMemberCategory = memberCategoryRepository.findByMemberAndCategory(member, category);
        assertThat(findedMemberCategory).isEmpty();
```
`P2` : 가져오는 주체가 `MemberCategory`이기에 이를 더 드러내보는건 어떨까요?